### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by rushikeshacharya**
### What I did
I have made the changes in the sqrt function as suggested in PR #1434.
### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: docs/built-in-functions.rst [hunk 1]
```diff
@@ -181,15 +181,15 @@ Returns ``sha256`` (SHA2 256bit output) hash of input.
 --------
 ::
 
-  def sqrt(a: decimal) -> decimal:
+  def sqrt(a: decimal) -> b:
```
Comment (fubuloubu) [inline docs/built-in-functions.rst]: This one is actually okay as is. Here `decimal` is the type that `sqrt` returns, according to it's function signature.

Comment (fubuloubu) [inline docs/built-in-functions.rst]:
```suggestion
  def sqrt(a: decimal) -> decimal:
```

File: docs/built-in-functions.rst [hunk 2]
```diff
@@ -181,15 +181,15 @@ Returns ``sha256`` (SHA2 256bit output) hash of input.
...
     """
     :param a:
     :type a: decimal, larger than 0.0
 
-    :output sqrt: decimal
+    :output b: decimal
```
Comment (fubuloubu) [inline docs/built-in-functions.rst]:
```suggestion
    :output sqrt: decimal
```

File: docs/built-in-functions.rst (hunk 1 above)
Comment (fubuloubu) [reply docs/built-in-functions.rst]: reverting this change, was fine as is

File: docs/built-in-functions.rst (hunk 2 above)
Comment (fubuloubu) [reply docs/built-in-functions.rst]: reverting this change, was fine as is

**Comment by davesque** [comment]
@fubuloubu I'm looking through this documentation article and the format is a bit confusing.  The article uses a kind of ad hoc docstring style that mimics the sphinx docstring format but doesn't really adhere to it.  It also specifies each function as an inline code block.  Seems like we should be using the sphinx reStructureText API as seen [here](http://www.sphinx-doc.org/en/stable/usage/quickstart.html#documenting-objects) to just get nicely formatted, linkable entries in the documentation.

**Comment by fubuloubu** [comment]
I think that's a good point. Can you add an issue to track this docs restructuring? (no pun intended)

**Comment by fubuloubu** [comment]
This PR no longer applies

**Pull request closed without merging**